src/sentry/tasks/store.py

```diff
@@ -31,6 +31,12 @@
 # Is reprocessing on or off by default?
 REPROCESSING_DEFAULT = False
 
+SYMBOLICATE_EVENT_APM_SAMPLING = float(
+    getattr(settings, "SENTRY_SYMBOLICATE_EVENT_APM_SAMPLING", 0.0)
+)
+
+SYMBOLICATOR_MAX_RETRY_AFTER = float(getattr(settings, "SYMBOLICATOR_MAX_RETRY_AFTER", 5))
```

Comment:
Why are we not using `settings.SYMBOLICATOR_MAX_RETRY_AFTER` directly but going through `getattr`?

Reply:
Just because it's like this all over the place and we are not consistent.

I believe the idea is that someone might be missing the setting and with `gettattr` and default value we prevent failures in such a case but imo it is safe here as we ship the defaults in `server.py`.